libs/openFrameworks/utils/ofFileUtils.cpp

```diff
@@ -1927,97 +1944,23 @@ void ofSetDataPathRoot(const of::filesystem::path& newRoot){
 }
 
 //--------------------------------------------------
-// MARK: - near future
 //of::filesystem::path ofToDataPath(const of::filesystem::path & path, bool makeAbsolute){
 std::string ofToDataPath(const of::filesystem::path & path, bool makeAbsolute){
```

Comment:
Looks like makeAbsolute is being ignored here?

Reply:
note: in the new implementation

Reply:
yes it is now returning always absolute but it is a easy change to return relative if needed.. I want first discuss the general idea of having a different current_path pointing to data, and I can implement this back if needed.

Reply:
I've fixed the makeAbsolute boolean to return rel or abs accordingly.